src/main/java/de/retest/recheck/RecheckImpl.java

```diff
@@ -90,6 +97,9 @@ public void check( final Object toVerify, final RecheckAdapter adapter, final St
 
 	protected ActionReplayResult createActionReplayResult( final Object toVerify, final RecheckAdapter adapter,
 			final String currentStep ) {
+		final DefaultValueFinder defaultFinder = adapter.getDefaultValueFinder();
+		usedFinders.put( currentStep, defaultFinder );
```

Comment:
If you want to prevent overwriting of the value. But if the API is used correctly, there should not be multiple calls of this method with the same `currentStep`.